entities/config.go

```diff
@@ -12,29 +12,21 @@ type Client struct {
 	ID    string `toml:"user_id"`
 }
 
-type ServerConfig_t struct {
-	MotherShipIP   string            `toml:"motherShipIP"`
-	MotherShipPort string            `toml:"motherShipPort"`
-	MotherShipSSL  bool              `toml:"motherShipSSL"`
+type Server struct {
 	Accounts       []ServerAccount_t `toml:"accounts"`
 }
 
 type ServerAccount_t struct {
-	UserName string `toml:"userName"`
-	// FIXME: change to "ID"
-	Nonce    string `toml:"nonce"`
-	Registed bool   `toml:"registered"`
-}
-
-type ServerToml_t struct {
-	Server ServerConfig_t `toml:"server"`
+	Name       string `toml:"user_name"`
```

Comment:
Ist camel case toml standard?

Reply:
There is no standard, but on toml.io they use snake case